### PR TITLE
add NEON RGB camera catalog

### PR DESCRIPTION
**Pull request by bhass-neon**
Add catalog and examples for NEON RGB camera imagery

### CODE REVIEW

File: catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_RGB_001.jsonnet [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+      'Phase One D8900',
+      'Phase One IQ180',
+      'Phase One iX-RS 1000',
+      'Phase One iXM-RS 150F',
+    ],
+    'eo:bands': [
+      {
+        name: 'R',
+        description: 'Red',
+        'gee:units': units.dimensionless,
```
Comment (simonff) [inline catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_RGB_001.jsonnet:166]: for NAIP, we use units.dn

https://github.com/google/earthengine-catalog/blob/main/catalog/USDA/USDA_NAIP_DOQQ.jsonnet

File: catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_RGB_001.jsonnet [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    // TODO(bhass-neon): Add doi link.
+    // {
+    //   rel: ee_const.rel.cite_as,
+    //   href: '',
+    //   type: ee_const.media_type.html,
+    // },
+  ],
+  keywords: [
+#    'airborne',
+#    'ecology',
```
Comment (simonff) [inline catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_RGB_001.jsonnet:63]: could you remove 'ecology'?

File: examples/NEON/NEON_RGB.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+var camera = ee.ImageCollection(
```
Comment (simonff) [inline examples/NEON/NEON_RGB.js:1]: you can remove the JS files under NEON

**Comment by bhass-neon** [comment]
This should be ready now! All the suggestions were applied in the last 2 commits.

**Review by dpencosk — APPROVED** [review]

**Pull request merged**